### ISSUE TITLE
small improvements for functorial composition

### ISSUE THREAD

**Issue by mantepse** [sagemath/sage#34480]
<div id="comment:0"></div>

It would be nice to have more applications for this method.

Depends on #34422

CC:  @tscrim

Component: **combinatorics**

Keywords: **LazyPowerSeries**

Author: **Martin Rubey**

Branch/Commit: **[u/mantepse/small_improvements_for_functorial_composition](https://github.com/sagemath/sagetrac-mirror/tree/u/mantepse/small_improvements_for_functorial_composition) @ [`c65bd48`](https://github.com/sagemath/sagetrac-mirror/commit/c65bd48952b6bccbf55273956e2660ac322a4fb8)**

_Issue created by migration from https://trac.sagemath.org/ticket/34480_

**Comment by mantepse** [comment]
Description changed:
``````diff
--- 
+++ 
@@ -1 +1 @@
-
+It would be nice to have more applications for this method.
``````
